tests/integration/test_get_analyses.py

```diff
@@ -0,0 +1,112 @@
+from http import HTTPStatus
+from flask.testing import FlaskClient
+from trailblazer.constants import Pipeline, TrailblazerStatus
+
+from trailblazer.store.models import Analysis
+
+
+def test_get_analyses(client: FlaskClient, analyses: list[Analysis]):
+    # GIVEN analyses
+
+    # WHEN retrieving all analyses
+    response = client.get(f"/api/v1/analyses?pageSize={len(analyses)}")
+
+    # THEN it gives a success response
+    assert response.status_code == HTTPStatus.OK
+
+    # THEN it should return all analyses
+    assert len(response.json["analyses"]) == len(analyses)
+
+
+def test_get_analyses_sorted_by_ticket_id(client: FlaskClient, analyses: list[Analysis]):
+    # GIVEN analyses
+
+    # WHEN retrieving analyses sorted by ticket id
+    response = client.get(
+        f"/api/v1/analyses?pageSize={len(analyses)}&sortField=ticket_id&sortOrder=asc"
+    )
+
+    # THEN it gives a success response
+    assert response.status_code == HTTPStatus.OK
+
+    # THEN it should return all analyses
+    assert len(response.json["analyses"]) == len(analyses)
+
+    # THEN the analyses should be sorted by the ticket id
+    ticket_ids = [analysis["ticket_id"] for analysis in response.json["analyses"]]
+    assert ticket_ids == sorted(ticket_ids)
+
+
+def test_get_analyses_filtered_by_single_status(client: FlaskClient, analyses: list[Analysis]):
+    # GIVEN analyses with different statuses
+
+    # WHEN retrieving completed analyses
+    response = client.get("/api/v1/analyses?status[]=completed")
+
+    # THEN it gives a success response
+    assert response.status_code == HTTPStatus.OK
+
+    # THEN it should return all analyses with the status completed
+    completed_analyses = [a for a in analyses if a.status == TrailblazerStatus.COMPLETED]
```

Comment:
I agree in general

Reply:
But this is a test and I think repeating analysis 4 times on one row is redundant

Reply:
It sets a bad patter that can spread and one having clean code in tests is arguably equally import as the rest of the code (speaking from having spent hours refactoring bad practices in this test suite). Would rather not go there again if it can be avoided.

Reply:
Sure 👍 I think list concatenation is a special case but I'll rename it